### PR TITLE
RDM-7781: CCD ITHC: Verbose Error Messages

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDM-7781

### Change description ###

Returned a custom error message to the caller

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/DocumentsOperation.java [hunk 1]
```diff
@@ -69,11 +69,11 @@ public DocumentsOperation(final SecurityUtils securityUtils,
...
         } catch (Exception e) {
             LOG.error(String.format(
-                "Cannot get documents for the Jurisdiction:%s, Case Type Id:%s, Case Reference:%s",
-                jurisdictionId, caseTypeId, caseReference), e);
+                "Cannot get documents for the Jurisdiction:%s, Case Type Id:%s, Case Reference:%s, because of %s",
+                jurisdictionId, caseTypeId, caseReference, e.getMessage()), e);
             throw new ServiceException(String.format("Cannot get documents for the Jurisdiction:%s, Case Type Id:%s, "
                     + "Case Reference:%s, because of %s",
-                jurisdictionId, caseTypeId, caseReference, e));
+                jurisdictionId, caseTypeId, caseReference, e.getMessage()), e);
```
Comment (MSancaktutar) [inline src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/DocumentsOperation.java:76]: Ideally, even the message inside Exception should not be communicated to the client.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Pls see comment.

File: src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/DocumentsOperation.java (hunk 1 above)
Comment (rajaram-nagarajan) [reply src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/DocumentsOperation.java]: Fixed

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request closed without merging**